Source/WebCore/Modules/applepay/ApplePaySessionPaymentRequest.h

```diff
@@ -184,6 +184,11 @@ class ApplePaySessionPaymentRequest {
     void setMerchantCategoryCode(const String& merchantCategoryCode) { m_merchantCategoryCode = merchantCategoryCode; }
 #endif
 
+#if ENABLE(APPLE_PAY_DELEGATED_REQUEST)
+    const std::optional<bool> isDelegatedRequest() const { return m_isDelegatedRequest; }
```

Comment:
A const lvalue return type is not super helpful, it should either return a reference or just a plain regular copy.


```suggestion
   std::optional<bool> isDelegatedRequest() const { return m_isDelegatedRequest; }
```

or

```suggestion
    const std::optional<bool> & isDelegatedRequest() const { return m_isDelegatedRequest; }
```

I would go with the former.